src/EventListener/DoctrineTaskExecutionListener.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of php-task library.
+ *
+ * (c) php-task
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Task\TaskBundle\EventListener;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Task\Event\TaskExecutionEvent;
+
+/**
+ * Listens on task-execution events.
+ */
+class DoctrineTaskExecutionListener
```

Comment:
Sorry to mention that again, but I missed it the last time... Wouldn't it be better to make it a subscriber? Then it could handle all the doctrine specific settings, which might be added later. Otherwise this would have to be renamed to something to include `clear`.